.github/workflows/tests.yaml

```diff
@@ -8,9 +8,6 @@ on:
 
 env:
     fail-fast: true
-    PHPUNIT_FLAGS: "-v"
```

Comment:
The PHPUnit 11 equivalent would be `--debug`, right?

Reply:
No, debug is much more verbose than the previous verbose flag (up to a point where using it by default wouldn't make much sense). Verbose is removed without replacement (https://github.com/sebastianbergmann/phpunit/issues/5647#issuecomment-1890483283)

Reply:
ah okay, thanks for the clarification

Reply:
The replacement is actually the set of separate `display-*` flags